octokit/activity_events.go

```diff
@@ -0,0 +1,73 @@
+package octokit
+
+import (
+	"time"
+)
+
+var (
+	PublicEventsURL = Hyperlink("/events")
+)
+
+func (c *Client) ActivityEvents() *ActivityEventsService {
+	return &ActivityEventsService{client: c}
+}
+
+// ActivityEventsService is a service providing access to event activity
+type ActivityEventsService struct {
+	client *Client
+}
+
+func (r *ActivityEventsService) One(uri *Hyperlink, params M) (event *Event,
```

Comment:
@dhruvsinghal: Upon closer look, I don't think this service has a `.One()` use case does it?